Cli-CredentialHelper/OperationArguments.cs

```diff
@@ -58,6 +58,7 @@ internal OperationArguments(TextReader stdin)
         public string CredUsername { get; private set; }
         public Interactivity Interactivity { get; set; }
         public bool PreserveCredentials { get; set; }
+
```

Comment:
Why a vertical white space here and not between `Interactivity` and `PreserveCredentials`?